### PR TITLE
Perfdash changes to display network performance benchmark results

### PR DESCRIPTION
**Pull request by hanamantagoudvk**
What type of PR is this?

/kind feature

What this PR does / why we need it:

This review is one of the changeset done as part of the enhancement/feature (Reference #1498)

This changeset has PerfDash changes for displaying the results in dashboard.

### CODE REVIEW

File: perfdash/config.go [hunk 1]
```diff
@@ -340,6 +340,136 @@ var (
 					Parser:           parsePerfData,
 				}},
 		},
+		"Network_Baseline": {
+			"TCP_P2P": []TestDescription{{
+				Name:             "standalone",
+				OutputFilePrefix: "NetworkPerformanceMetrics1:1TCPP2P",
```
Comment (wojtek-t) [inline perfdash/config.go:346]: Can we have better names (it's hard to parse it), e.g.:

NetworkPerformanceMetrics_1:1_TCP_P2P

[Yes - it requires small changes to the code generating them]

File: perfdash/config.go [hunk 2]
```diff
@@ -340,6 +340,136 @@ var (
...
+				Parser:           parsePerfData,
+			}},
+
+			"TCP_ClusterIP": []TestDescription{
+				{
+					// TODO(oxddr): remove this around Sep '19 when we stop showing old data
```
Comment (wojtek-t) [inline perfdash/config.go:352]: Please remove those TODOs - those are completely unrelated (it was for migration file names, which you're obviously nt doing).

File: perfdash/config.go (hunk 1 above)
Comment (wojtek-t) [reply perfdash/config.go:346]: Also - converge on the order (the the name should be the ame as in the file prefix)

Comment (VivekThrivikraman-est) [reply perfdash/config.go]: Modified

File: perfdash/config.go (hunk 2 above)
Comment (VivekThrivikraman-est) [reply perfdash/config.go]: It wasn't there in the final commit, something went wrong during splitting(old commit was pushed). Have reverted to the actual changes.

File: clusterloader2/testing/network/config.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 {{$NUMBER_OF_SERVERS := .CL2_NUMBER_OF_SERVERS}}
 {{$NUMBER_OF_CLIENTS := .CL2_NUMBER_OF_CLIENTS}}
 
-name: Network Performance
+name: network_performance
```
Comment (wojtek-t) [inline clusterloader2/testing/network/config.yaml:5]: Why do you change this?

File: perfdash/config.go [hunk 4]
```diff
@@ -340,6 +340,45 @@ var (
...
 		},
+		"Network_Baseline": {
+			"TCP_P2P": []TestDescription{{
+				Name:             "network_performance_tcp-1:1",
+				OutputFilePrefix: "NetworkPerformanceMetrics_1:1_TCP_P2P",
+				Parser:           parsePerfData,
+			}},
+
+			"UDP_P2P": []TestDescription{
+				{
```
Comment (wojtek-t) [inline perfdash/config.go:351]: Please make pagination consistent - just use the style you use above [i.e. {{ in the previous line]

same below

File: clusterloader2/testing/network/config.yaml (hunk 3 above)
Comment (VivekThrivikraman-est) [reply clusterloader2/testing/network/config.yaml:5]: 2 reasons : 1) Other tests in cluster loader where there are 2 words they are not using capital letters(eg node-throughput). But just noticed they use small letters and words connected by "-" rather than "_".Shall I change this also to have "-" instead of "_"?
2)The file generated would have this as part of it's name , and having space just for this would seem awkward.

Comment (wojtek-t) [reply clusterloader2/testing/network/config.yaml:5]: ok - that's fine

File: perfdash/config.go (hunk 4 above)
Comment (VivekThrivikraman-est) [reply perfdash/config.go]: Modified
